packages/wallets/src/features/accounts/modules/ManualService/ManualService.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import { useSettings } from '@deriv/api-v2';
+import { Loader } from '@deriv-com/ui';
+import { ModalStepWrapper } from '../../../../components';
+import { DocumentSelection } from './components';
+import { manualDocumentsMapper, TManualDocumentComponent, TManualDocumentType } from './utils';
+
+type TManualServiceProps = {
+    onCompletion?: VoidFunction;
+};
+
+type TSelectedManualDocument = keyof TManualDocumentType | undefined;
+
+const ManualService: React.FC<TManualServiceProps> = ({ onCompletion }) => {
+    const { data: accountSettings, isLoading: isAccountSettingsLoading } = useSettings();
+    const [selectedManualDocument, setSelectedManualDocument] = useState<TSelectedManualDocument>();
+    let SelectedDocument: TManualDocumentComponent;
```

Comment:
hey @aum-deriv 
Defining a component inside the definition of another component is react anti-pattern, because it can lead to unpredictable consequences when parent rerenders.
Instead try to lift up the component:
e.g 
```
const Document = ({ component: Component, country_code, onCompletion, resetSelectedDocument }) => {
    return (
        <Component
            documentIssuingCountryCode={country_code}
            onClickBack={resetSelectedDocument}
            onCompletion={onCompletion}
        />
    );
};
```